acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ThreadBesuNodeRunner.java

```diff
@@ -165,12 +164,7 @@ public void startNode(final BesuNode node) {
             .build();
 
     final RunnerBuilder runnerBuilder = new RunnerBuilder();
-    if (node.getPermissioningConfiguration().isPresent()) {
-      final PermissioningConfiguration permissioningConfiguration =
-          node.getPermissioningConfiguration().get();
-
-      runnerBuilder.permissioningConfiguration(permissioningConfiguration);
-    }
+    runnerBuilder.permissioningConfiguration(node.getPermissioningConfiguration());
```

Comment:
```suggestion
    node.getPermissioningConfiguration().ifPresent(runnerBuilder::permissioningConfiguration);
```

Reply:
It's optional all the way down! I think the problem is more that we're using the `ifPresent` of something to indicate that that particular feature is enabled in a few places. Previously it was doing something similar but in the consuming code it would wrap it again to an optional so just wanted to make it more explicit.

Reply:
Ah, now I see where it's being set as empty as well, so keeping the builder parameter as Optional makes sense.